webapp/src/components/StackedChartReport/index.js

```diff
@@ -0,0 +1,186 @@
+import React, { memo, useCallback } from 'react'
+import FileSaver from 'file-saver'
+import PropTypes from 'prop-types'
+import { useTranslation } from 'react-i18next'
+import { useCurrentPng } from 'recharts-to-png'
+import { makeStyles } from '@mui/styles'
+import { Box } from '@mui/system'
+import {
+  ComposedChart,
+  Bar,
+  XAxis,
+  YAxis,
+  CartesianGrid,
+  Tooltip,
+  Legend,
+  Line,
+  ResponsiveContainer
+} from 'recharts'
+
+import { formatWithThousandSeparator } from '../../utils'
+
+import styles from './styles'
+
+const useStyles = makeStyles(styles)
+
+const CustomTooltip = ({ payload = [], label = '', typeCurrency = '' }) => {
+  const { t } = useTranslation('generalForm')
+
+  return (
+    <div>
+      <strong>{label}</strong>
+      {payload &&
+        payload.map((data, i) => (
+          <div key={`${i}-tooltip`}>{`${
+            data.dataKey === 'EXCHANGE_RATE'
+              ? t('exchangeRate')
+              : data.dataKey === `${typeCurrency}_CLAIMED`
+              ? `${t('claimedCat')} `
+              : data.dataKey === `${typeCurrency}_UNCLAIMED`
+              ? `${t('unclaimedCat')} `
+              : data.dataKey === `${typeCurrency}_CATEGORIZED`
+              ? `${t('categorizedCat')} `
+              : data.dataKey === `${typeCurrency}_UNCATEGORIZED` &&
+                t('uncategorizedCat')
+          }: ${formatWithThousandSeparator(
+            data.payload[data.dataKey],
+            2
+          )}`}</div>
+        ))}
+    </div>
+  )
+}
+CustomTooltip.propTypes = {
```

Comment:
@tetogomez it should have a blank line before, shouldn't it?

Reply:
yep, add a blank space